lib/addDefaults.js

```diff
@@ -31,6 +32,12 @@ function addDefaults(gltf) {
     ForEach.mesh(gltf, function(mesh) {
         ForEach.meshPrimitive(mesh, function(primitive) {
             primitive.mode = defaultValue(primitive.mode, WebGLConstants.TRIANGLES);
+            if (!defined(primitive.material)) {
+                if (!defined(gltf.materials)) {
+                    gltf.materials = [];
+                }
+                primitive.material = addToArray(gltf.materials, {});
```

Comment:
Maybe give this a name, like "default"?That way it's not a completely empty object, and we'll be able to easily identify the default material added later.